### PR TITLE
Sprint2 3 searching in a list

### PR DESCRIPTION
**Pull request by angelbutterfly**
Hey Pacassy thanks for reviewing my code.

### CODE REVIEW

File: src/components/Dashboard.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <template>
   <div class="component component--dashboard">
     <h1>Dashboard</h1>
-    <p>Welcome to <em>trckr</em>!</p>
-    <p><router-link :to="{path: '/project/create'}">Create</router-link> a new project here!</p>
-
+    <p>Welcome to <em>trckr</em>!</p><br />
```
Comment (dpacassi) [inline src/components/Dashboard.vue]: We don't need a `<br />` here, structural distances should be defined via CSS.

File: src/components/Dashboard.vue [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+    <p><router-link :to="{path: '/project/create'}">Create</router-link> a new project here!</p><br />
```
Comment (dpacassi) [inline src/components/Dashboard.vue]: We don't need a `<br />` here, structural distances should be defined via CSS.

File: src/components/Dashboard.vue [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+    <input type="text" v-model="search" placeholder="search for a project"/>
```
Comment (dpacassi) [inline src/components/Dashboard.vue]: Capitalize the `s` in `search` and add a missing white space before the `/>`

File: src/components/SearchProject.vue [hunk 4]
```diff
@@ -0,0 +1,35 @@
+  <template>
```
Comment (dpacassi) [inline src/components/SearchProject.vue]: Do we need this file? I don't see it being used anywhere.

**Comment by angelbutterfly** [comment]
Thanks for your review. I've made all changes you've asked for and the code is ready for your review again.

**Pull request merged**